ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -271,21 +291,30 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
         self.animatesWhenPressed = animatesWhenPressed
         self.accessoryType = accessoryType
 
-        titleButtonLeadingImageView.image = titleImage
-        titleButtonLeadingImageView.isHidden = titleImage == nil
+        titleLeadingImageView.image = titleImage
+        titleLeadingImageView.isHidden = titleImage == nil
+
+        setupTitleLine(titleContainer, label: titleLabel, trailingImageView: titleTrailingImageView, text: title, interactive: interactivePart.contains(.title), accessoryType: accessoryType)
+        if titleLeadingImageView.image != nil {
```

Comment:
Does `titleLeadingImageView.image` change between lines 297 and 298, or could you just add it in `setupTitleLine`? The trailing image is added there, and it would be better to just add them all in one place, in order, instead of add two of them, and later insert one into the front.

Reply:
`titleLeadingImageView.image` doesn't change, but structuring it this way allows `setupTitleLine` to also be responsible for the initial clearing out of the stack views. Also, adding leading image view infrastructure to `setupTitleLine` results in a binary size increase of about 128 bytes.

At the end of the day, I don't think this should be a blocking change, so I'm gonna go with this as is.